k8s/preview/common/cnp/plum-frontend.yaml

```
@@ -0,0 +1,21 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: plum-frontend
+  namespace: cnp
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.nodejs: glob:prod-*
+spec:
+  releaseName: plum-frontend
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: plum-frontend
+    version: 0.0.3
+  values:
+    nodejs:
+      image: hmctspublic.azurecr.io/plum/frontend:prod-41c16310
+      ingressHost: plum.preview.platform.hmcts.net
```

this will not be automatically registered, lets use 
```suggestion
      ingressHost: plum-frontend.service.core-compute-preview.internal.
```